src/domObject.js

```diff
@@ -20,18 +20,28 @@ export default class UI {
 
   displayData(leadersArray = this.leadersArray) {
     const leadersList = document.querySelector('.leaders-list');
+    const listHeader = document.querySelector('.list-header');
```

Comment:
- [ **OPTIONAL** ] What do you think if you display players of your game by sorting them with ones with High score to those with low score(from top to bottom). This can help users to easily get their names from the list and your application will look again smarter.